ocs_ci/ocs/resources/mcg.py

```diff
@@ -773,10 +773,14 @@ def cli_create_bucketclass_over_namespacestores(
             if "ttl" in namespace_policy:
                 cmd += f" --ttl=={namespace_policy['ttl']}"
             self.exec_mcg_cmd(cmd)
+        elif namespace_policy_type == constants.NAMESPACE_POLICY_TYPE_MULTI.lower():
+            cmd += f" --read-resources='{namestores_name_str}'"
+            cmd += f" --write-resource='{namespacestores[0].name}'"
+            self.exec_mcg_cmd(cmd)
         else:
             raise NotImplementedError(
                 f"Cli creating of bucketclass on namespacestore "
-                f"with policy {namespace_policy_type} is not implemented yet"
+                f"with policy {namespace_policy_type} is supported"
```

Comment:
We should remove this exception since we implemented it

Reply:
I prefer to leave this exception in case unsupported policy type is passed but you are right, the wording of the exception is confusing and I change it to "not supported".